### PR TITLE
openflow_input: add decap TLV

### PR DESCRIPTION
**Pull request by rlane**
Reviewer: @sandipshah

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/181/
Test PASSed.

**Comment by sandipshah** [comment]
ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2015.10.01.1729-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2015.10.01.1729-m.loxigen.master/abat.log
/cc

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -552,3 +552,21 @@ struct of_bsn_tlv_ipv6 : of_bsn_tlv {
...
+    OFP_BSN_DECAP_VXLAN = 0,
+    OFP_BSN_DECAP_ERSPAN = 1,
+    OFP_BSN_DECAP_L2_GRE = 2,
+    OFP_BSN_DECAP_NVGRE = 3,
+    OFP_BSN_DECAP_CAPWAP = 4,
+    OFP_BSN_DECAP_L2_MPLS = 5,
+    OFP_BSN_DECAP_L3_GRE = 6,
+    OFP_BSN_DECAP_GTP = 7,
+    OFP_BSN_DECAP_L3_MPLS = 8,
+};
```
Comment (sandipshah) [inline openflow_input/bsn_tlv]: If I need to configure more than one decap on same service interface, I need to send multiple TLVs. Isn't bit mask a better option?

Comment (rlane) [inline openflow_input/bsn_tlv]: No, just send multiple TLVs.

**Pull request merged**